.github/workflows/update-tilesets.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow runs on sundays at 23:00
+name: "Pull Tileset Updates"
+
+on:
+  schedule:
+    - cron: "0 23 * * sun"
+
+jobs:
+  pull-updates:
+    if: github.repository == 'CleverRaven/Cataclysm-DDA'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get current timestamp
+        id: get-timestamp
+        uses: nanzm/get-time-action@v1.1
+        with:
+          format: 'YYYY-MM-DD'
+      - name: "Update Tilesets"
+        run: |
+          curl -s https://api.github.com/repos/I-am-Erk/CDDA-Tilesets/releases/latest | grep "browser_download_url.*zip" | cut -d : -f 2,3 | tr -d \" | wget -P gfx -i -
+          find gfx/ -name "*.zip" | xargs -I{} unzip -o {} -d gfx/
+          rm -fr gfx/*.zip
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: |
+            Routine Tileset updates on ${{ steps.get-timestamp.outputs.time }}
+
+            Co-authored-by: Fris0uman <41293484+Fris0uman@users.noreply.github.com> 
```

Comment:
```suggestion
```
It doesn't make sense to list me as a co-author, we can add link to the tileset repos in `tileset.txt` of tilesets that don't have it yet if we want to double down on proper attribution, but the update itself can be authored just by a github action.